### DISCUSSION TITLE
Feature request: add and edit description on Replay

### DISCUSSION THREAD

**Discussion by jonbell-lot23** [RecordReplay/devtools#2825] [Q&A] [answered]
Split out from this item:
https://github.com/RecordReplay/devtools/issues/2740

**Comment by jasonLaster** [comment]
Sorry for the confusion. The goal of the GTM doc was to add a summary section. Lets use #2740 and think about it from a user perspective

**Comment by jonbell-lot23** [comment]
Happy to remove the GTM issue tag, but would like to track this as its own item apart from the summary item which we agree is GTM material.

**Comment by jonbell-lot23** [comment]
Converted to discussion

**Comment by jonbell-lot23** [comment, marked as answer]
This has been fixed. @jcmorrow can you mark this as the answer? :)

**Reply by jcmorrow** [reply]
Actually I can't! 🤔
